fe1-web/model/objects/__tests__/Channel.test.ts

```diff
@@ -1,11 +1,43 @@
 import 'jest-extended';
 
+import { KeyPairStore } from 'store';
 import { Hash } from '../Hash';
-import { channelFromIds } from '../Channel';
+import {
+  channelFromIds,
+  generalChirpsChannel,
+  getLastChannel,
+  userSocialChannel,
+} from '../Channel';
+import { KeyPair } from '../KeyPair';
+import { PublicKey } from '../PublicKey';
 
-test('Channel should return channels properly', () => {
-  const id: Hash = Hash.fromStringArray('id');
+const FAKE_ID = Hash.fromStringArray('id');
+const PUBLIC_KEY = '1234';
+const PRIVATE_KEY = '5678';
 
-  expect(channelFromIds()).toEqual('/root');
-  expect(channelFromIds(id)).toEqual(`/root/${id}`);
+describe('Channel object', () => {
+  it('channelFromIds should should return the correct channel', () => {
+    expect(channelFromIds()).toEqual('/root');
+    expect(channelFromIds(FAKE_ID)).toStrictEqual(`/root/${FAKE_ID}`);
+  });
+
+  it('userSocialChannel should return the correct channel', () => {
+    const keyPair = KeyPair.fromState({
+      publicKey: PUBLIC_KEY,
+      privateKey: PRIVATE_KEY,
+    });
+    KeyPairStore.store(keyPair);
```

Comment:
I don't remember: is there a global cleanup of the stores across tests?
Or have you just permanently modified the keyPair for all the other tests? 
Ideally, tests should not affect each other.
If there's no global cleanup, it might make sense to either:
1. look into it :-)  or 
2. ensure that the store contains the same information before and after the tests.

Reply:
From what I've seen, I think that each test that uses the KeyPairStore stores his own key pair in the beforeEach function, so it is overwritten before each test. Is it a problem ?

Reply:
Sounds okay, if that's what's happening.